ui/templates.go

```diff
@@ -9,7 +11,7 @@
     </head>
     <body>
         <!-- Your source after making -->
-        <script src="/script.js"></script>
-        <script>Elm.Main.embed(document.body)</script>
+        <script src="{{ .ExternalURL }}/script.js"></script>
```

Comment:
I believe this will still be needed for the simple reverse proxy situation I tested using nginx